jopa-integration-tests-jena/src/test/java/cz/cvut/kbss/jopa/test/integration/jena/MultipleInheritanceContextTest.java

```diff
@@ -0,0 +1,15 @@
+package cz.cvut.kbss.jopa.test.integration.jena;
+
+import cz.cvut.kbss.jopa.test.environment.JenaDataAccessor;
+import cz.cvut.kbss.jopa.test.environment.JenaPersistenceFactory;
+import cz.cvut.kbss.jopa.test.runner.MultipleInheritanceTestRunner;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MultipleInheritanceContextTest  extends MultipleInheritanceTestRunner {
```

Comment:
Why _Context_? Given the naming conventions between runners and tests, this class should be called `MultipleInheritanceTest`.

Reply:
SOLVED - renamed